mathics/builtin/system.py

```diff
@@ -871,3 +871,29 @@ def eval_with_symbol(self, symbol, evaluation: Evaluation) -> Integer:
         else:
             gc.collect()
             return Integer0
+
+
+class Breakpoint(Builtin):
+    """
+    ## <url>:trace native symbol:</url>
+
+    <dl>
+      <dt>'Breakpoint[]'
+      <dd> Invoke a Python breakpoint. By default, the Python debugger \
+           (pdb) is loaded. For loading other debuggers, the Python environment \
+           variable `PYTHONBREAKPOINT` can be utilized.
+    </dl>
+
+    >> (* Test with a disabled breakpoint. *)
+    >> SetEnvironment["PYTHONBREAKPOINT" -> "mathics.disabled_breakpoint"];
+    >> Breakpoint[]
+    Hit disabled breakpoint.
```

Comment:
Do you mean this?

```suggestion
    ...
```

As I suggested earlier, it's much simpler if you could please use the +/- button and the `suggestion` tag to indicate the exact change.

Reply:
I tried adding the +/- . Maybe the suggestion tag I missed. 

Here is another suggestion that I made a while ago.  Make this a branch of this repository. Then when folks want to make suggestions we can just make a branch off of your branch it has all of the changes and you can review and or commit on top.  This is the way we've been working on other PRs. 

```
-   Hit disabled breakpoint.
+   ...
...

No, neither is correct.  _Either_ would be okay of you add `= ` beforehand.  For example: 

```
= Hit disabled breakpoint.
```

See [line 484](https://github.com/Mathics3/mathics- lcore/blob/fb6fadaf486c83a7e73403f94d73ec7bc208fb76/mathics/builtin/system.py#L484) and [line 362](https://github.com/Mathics3/mathics-core/blob/fb6fadaf486c83a7e73403f94d73ec7bc208fb76/mathics/builtin/system.py#L362) if this file for examples of each kind of method.  

Also look at the previous read-the-docs link. If there is something unclear about that, then ask we can make this more clear.

Reply:
I see. I've edited my comment above and now it should likely reflect the expected changes. I looked at the read-the-docs link, and also the previous examples. Unfortunately, I missed this (some would say rather obvious) detail. Perhaps it would have helped to have an example following the table in read-the-docs.

Regarding the review suggestions, please click on the highlighted +/- button below, and then change the line within the `suggestion` block as desired. Then, if you click on the "Preview" tab, GitHub will already show the diff. This diff can be commit-ed either as a single commit, or all comment fixes as a "batch commit."

![](https://docs.github.com/assets/cb-26289/mw-1440/images/help/pull_requests/suggestion-block.webp)